packages/manager/apps/pci-instances/src/pages/instances/instance/dashboard/components/network/ExpandedNetwork.component.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC, useState } from 'react';
+import { Button, Icon } from '@ovhcloud/ods-react';
+import { useTranslation } from 'react-i18next';
+import NetworkItem from './NetworkItem.component';
+import { TInstanceAddress } from '@/types/instance/entity.type';
+
+const ExpandedNetwork: FC<{ networks: TInstanceAddress[] }> = ({
+  networks,
+}) => {
+  const { t } = useTranslation('common');
+  const [isExpand, setIsExpand] = useState(false);
+
+  const handleExpand = () => setIsExpand(!isExpand);
```

Comment:
```suggestion
  const handleExpand = () => setIsExpand(prev => !prev);
```

It's better to use the lambda with the previous value instead of the exposedValue (isExpand).